README.md

```diff
@@ -112,7 +112,7 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 - [Find broken links, missing images, etc in your HTML](https://github.com/peaceiris/actions-broken-link-checker)
 - [Fast website link checker in Go](https://github.com/peaceiris/actions-muffet)
 - [Fast link checker for Markdown and HTML in Go)](https://github.com/peaceiris/actions-liche)
-- [Send your code coverage to codecov.io](https://github.com/atrox/codecov-action)
+- [Send your code coverage to codecov.io](https://github.com/codecov/codecov-action)
```

Comment:
@ibrahim0814 I can't say that I have used either of these actions, so I can't answer this question, but is there anything "wrong" with the action from @Atrox Is there a reason that there shouldn't be two entries in the list for codecov actions?

Reply:
I actually think it is better to have the official one listed. Makes only sense. I only created the action because there was no official one yet. 

I’ll add a notice to my repo.

Reply:
@Atrox thanks for confirming, and for making a change to your repository.  Based on your approval, I am happy to merge this in.